kroxylicious-app/src/assembly/kroxylicious-start.sh

```diff
@@ -20,6 +20,7 @@ classpath() {
   echo "${class_path}"
 }
 
+export JAVA_OPTIONS="${JAVA_OPTIONS:-} -Dlog4j2.configurationFile=$(script_dir)/../config/log4j2.yml"
```

Comment:
Any idea what happens if the log4j config is specified twice?

Reply:
good point, the rightmost one appears to win, it's not really a specified behaviour. We should probably offer another environment variable like `KROXYLICIOUS_LOG4J2_CONFIG` to override it.

Reply:
That only half fixes the problem, it can still be specified multiple times but I guess much less likely. 

It's probably good enough for our needs.

Reply:
I'll switch the order so our specializations are leftmost

Reply:
or maybe we should not force this to be an arg, give the user a way to override it

Reply:
so change from `KROXYLICIOUS_LOG4J2_CONFIG` to `KROXYLICIOUS_JAVA_OPTIONS` so the user could set it to empty if required